LayoutTests/platform/ios/TestExpectations

```diff
@@ -27,6 +27,22 @@ fast/events/ios [ Pass ]
 fast/events/ios/ipad [ Skip ]
 fast/forms/ios [ Pass ]
 fast/forms/ios/ipad [ Skip ]
+fast/forms/switch [ Pass ]
+
+# More work required for these tests to run on iOS
+fast/forms/switch/click-animation-disabled.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-interrupted-document-removed.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-interrupted-type-change.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-interrupted.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-preventdefault.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-redundant-checked.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-redundant-disabled.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-twice-fast.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation-twice.html [ ImageOnlyFailure ]
+fast/forms/switch/click-animation.html [ ImageOnlyFailure ]
+fast/forms/switch/pointer-tracking-there-and-back-again-rtl.html [ Skip ]
+fast/forms/switch/pointer-tracking-there-and-back-again.html [ Skip ]
```

Comment:
Is it intentional that these two tests aren't also being updated to use `UIHelper` in this patch?

Reply:
Yeah, UIHelper doesn't have helpers for these as far as I can tell.

Reply:
I think you can rewrite those tests to use `UIHelper.EventStreamBuilder`.

Example: https://github.com/WebKit/WebKit/blob/main/LayoutTests/fast/events/touch/ios/scroll-on-touch-start-with-slow-event-listener.html

Reply:
I have written this code, but I can only get the tests to sometimes pass on a build with painting code. It's rather non-deterministic. Would you mind having a look and see if there's anything obviously wrong? I updated this PR.

Reply:
You need to use `async`/`await` whenever using `UIHelper` with user interaction.

Reply:
Why? I have the click event to tell me when the interaction ended, no?